g1_map_robot.cydsn/ZumoLibrary/Lidar.c

```diff
@@ -0,0 +1,39 @@
+#include "Lidar.h"
+#include "stdint.h"
+
+static int old_dist = -1;
+
+void
+Lidar_start()
+{
+  UART_3_Start();
+  Lidar_get_distance();
```

Comment:
Maybe an assertion here? Otherwise this line is useless? 

Something like:
`assert(getdist() > 0);`?

Reply:
Should be left as is, since the first transaction usually fails. It is done in order to have valid value in the buffer next time. (Even though it will be old)